### PR TITLE
Inherit function config from project config

### PR DESCRIPTION
**Pull request by mthenw**
I was expecting much more code but turns out that it's enough to have default values and support configs inheritance.

### CODE REVIEW

**Comment by mthenw** [comment]
I think it solves https://github.com/apex/apex/issues/52.

File: project/project.go [hunk 1]
```diff
@@ -16,6 +16,13 @@ import (
 	"github.com/tj/go-sync/semaphore"
 )
 
+const (
+	// Memory defines default memory value for every function in a project
+	Memory = 128
```
Comment (tj) [inline project/project.go]: `DefaultMemory` would be rad but other than that LGTM!

Comment (mthenw) [inline project/project.go]: Done

**Comment by tj** [comment]
boom!

**Pull request merged**